easybuild/easyconfigs/c/ColabFold/ColabFold-1.5.5-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,187 @@
+        ],
+    }),
+    ('colabfold', version, {
+        'patches': ['%(name)s-%(version)s_relax_requirements.patch'],
+        'preinstallopts': """sed -i 's/^\\(python =.*\\)<=3.10"$/,<=%(pyshortver)s"/;""" +
+                          """s/flow-cpu/flow/' pyproject.toml &&""",
+        'source_urls': ['https://github.com/%(github_account)s/%(name)s/archive'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'use_pip_extras': 'alphafold',
+        'checksums': [
+            {'colabfold-1.5.5.tar.gz': '7fb0ad263c24ec00d46f2dff1611989234bc1351d71653dd06006f2c88f83dcb'},
+            {'colabfold-1.5.5_relax_requirements.patch':
+             'bcbcd7f403fea2e98ea04568d6ef358c72ad4beab762e701903473abf1883d03'},
+        ],
+    }),
+]
+
+_alphafold_buildir = '%%(builddir)s/AlphaFold/alphafold-%s' % _sokrypton_commit
+postinstallcmds = [
+    "mkdir -p %(installdir)s/bin",
+    "cp %s/run_alphafold.py %%(installdir)s/bin/run_alphafold.py" % _alphafold_buildir,
+    "chmod a+x %(installdir)s/bin/run_alphafold*.py",
+    "cd %(installdir)s/bin && ln -s run_alphafold.py alphafold",
+    "cp -a %s/scripts %%(installdir)s/" % _alphafold_buildir,
+    # Copy notebooks to installation directory
+    "mkdir %(installdir)s/notebooks",
+    "cp AlphaFold2.ipynb ESMFold.ipynb RoseTTAFold.ipynb %(installdir)s/notebooks/",
+]
+
+sanity_check_paths = {
+    'files': [
+        'bin/run_alphafold.py', 'bin/colabfold_batch', 'bin/colabfold_search', 'bin/colabfold_split_msas',
+        'lib/python%(pyshortver)s/site-packages/alphafold/common/stereo_chemical_props.txt'
+    ],
+    'dirs': ['lib/python%(pyshortver)s/site-packages', 'notebooks'],
+}
+
+sanity_check_commands = [
+    "pdbfixer --help",
+    "run_alphafold.py --help 2>&1 | grep 'Full AlphaFold protein structure prediction script'",
+    "colabfold_batch --help",
+]
+
+sanity_pip_check = True
+
+modextravars = {
+    'TF_FORCE_UNIFIED_MEMORY': '1',
+    'XLA_PYTHON_CLIENT_MEM_FRACTION': '3',
+    # 'ALPHAFOLD_DATA_DIR': '/path/to/AlphaFold_DBs/2.3.2',  # please adapt
+    'ALPHAFOLD_DATA_DIR': '/scratch/AlphaFold_DBs/2.3.2',  # please adapt,
+    'OPENMM_RELAX': 'CUDA',  # unset or set to 'CPU' in order not to run the energy minimization on GPU; PR#189
+    # Adapt in order to use a different version of UniRef30 by default,
+    # e.g.,  v2023_02 from https://wwwuser.gwdg.de/~compbiol/uniclust/2023_02/UniRef30_2023_02_hhsuite.tar.gz:
+    'ALPHAFOLD_UNIREF30_VER': '2021_03',
+}
+
+modloadmsg = """
+%(name)s notebooks are located in $EBROOTCOLABFOLD/notebooks
+"""
+
+postinstallmsgs = [
+    "A newer version of UniRef30 (2023_02) is available at: "
+    "https://wwwuser.gwdg.de/~compbiol/uniclust/2023_02/UniRef30_2023_02_hhsuite.tar.gz. "
+    "Untar to $ALPHAFOLD_DATA_DIR/uniref30/ and set the default version accordingly by changing "
+    "modextravars:ALPHAFOLD_UNIREF30_VER."
```

Comment:
`modextravars` is meaningless for end users, I would rephrase this to talk about changing the "`$ALPHAFOLD_UNIREF30_VER` environment variable" ?

Reply:
@boegel Do "end users" get this message? I assumed `postinstallmsgs` is intended for admins.